gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/EncounterMapperTest.java

```diff
@@ -97,18 +97,19 @@
     private ConsultationListMapper consultationListMapper;
 
     @Mock
-    private ImmunizationChecker immunizationChecker;
+    private DatabaseImmunizationChecker immunizationChecker;
 
-    @InjectMocks
```

Comment:
You could have `@InjectMocks` over EncounterMapper, and ResourceReferenceUtil as a mock